commands/user_e2e_test.go

```diff
@@ -136,3 +140,150 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+
+func (s *MmctlE2ETestSuite) TestDeleteUsersCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("Delete user", func(c client.Client) {
+		printer.Clean()
+
+		previousVal := s.th.App.Config().ServiceSettings.EnableAPIUserDeletion
+		s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableAPIUserDeletion = true })
+		defer s.th.App.UpdateConfig(func(cfg *model.Config) { *cfg.ServiceSettings.EnableAPIUserDeletion = *previousVal })
+
+		cmd := &cobra.Command{}
+		confirm := true
+		cmd.Flags().BoolVar(&confirm, "confirm",confirm,"confirm")
+
+		newUser, appErr := s.th.CreateUser()
```

Comment:
The test helper `CreateUser` only returns a `*model.User`. This is why CI is failing.

Reply:
adjusted, changed function to correct one, thanks @ashishbhate